requirements.in

```diff
@@ -9,4 +9,4 @@ itsdangerous==1.1.0
 digitalmarketplace-apiclient
 digitalmarketplace-content-loader
 digitalmarketplace-utils>=57.0.0
-git+https://github.com/alphagov/govuk-frontend-jinja.git@v0.5.2-alpha#egg=govuk-frontend-jinja==0.5.2-alpha
+git+https://github.com/alphagov/govuk-frontend-jinja.git@v0.5.8-alpha#egg=govuk-frontend-jinja==0.5.8-alpha
```

Comment:
reminds me this is another one on the list to move to NPM! Not for this PR though!

Reply:
Can't sadly, (at least not with this name), as there's already a govuk-frontend-jinja on PyPI